### PR TITLE
EZP-29767: Added overflow to popup when it's partially hidden

### PR DESCRIPTION
**Pull request by GrabowskiM**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-29767
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no

Added overflow and max-height to popup when it's partially hidden

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/modules/common/tooltip-popup/tooltip.popup.component.js [hunk 1]
```diff
@@ -1,13 +1,37 @@
...
+
+    useLayoutEffect(() => {
+        const { top, height } = contentRef.current.getBoundingClientRect();
+
+        if (top < HEADER_HEIGHT) {
+            setMaxHeight(height + top - HEADER_HEIGHT);
+        } else if (top > HEADER_HEIGHT) {
+            setMaxHeight(INITIAL_HEIGHT);
+        }
+
```
Comment (dew326) [inline src/modules/common/tooltip-popup/tooltip.popup.component.js]:
```suggestion
```

File: src/modules/common/tooltip-popup/tooltip.popup.component.js [hunk 2]
```diff
@@ -1,13 +1,37 @@
...
+    });
+
     const attrs = {
         className: 'c-tooltip-popup',
         hidden: !props.visible,
     };
```
Comment (dew326) [inline src/modules/common/tooltip-popup/tooltip.popup.component.js]:
```suggestion
```

**Review by dew326 — APPROVED** [review]
Just small CS

**Review by OstafinL — APPROVED** [review]

**Review by katarzynazawada — CHANGES_REQUESTED** [review]
React error occurs on Firefox when opening popup. http://g.recordit.co/QmH8NeTPp3.gif

**Review by dew326 — APPROVED** [review]

**Review by OstafinL — APPROVED** [review]

**Review by katarzynazawada — APPROVED** [review]

**Pull request merged**